server/tests/api/videos/video-transcoder.ts

```diff
@@ -91,6 +93,93 @@ describe('Test video transcoding', function () {
     expect(torrent.files[0].path).match(/\.mp4$/)
   })
 
+  it('Should transcode high bit rate mp3 to proper bit rate', async function () {
+    this.timeout(60000)
+
+    console.log('is streamable?')
+
+    const videoAttributes = {
+      name: 'my super name for server 2',
+      description: 'my super description for server 2',
+      fixture: 'video_short_mp3_256k.mp4'
+    }
+    const fixturePath = buildAbsoluteFixturePath(videoAttributes.fixture)
+    const isFixureStreamable = await isVideoFileStreamable(fixturePath)
+    expect(isFixureStreamable.isAAC).to.be.false
+
+    console.log('is streamable.')
+    await uploadVideo(servers[1].url, servers[1].accessToken, videoAttributes)
+
+    await waitJobs(servers)
+
+    const res = await getVideosList(servers[1].url)
+
+    const video = res.body.data[0]
+    const res2 = await getVideo(servers[1].url, video.id)
+    const videoDetails: VideoDetails = res2.body
+
+    expect(videoDetails.files).to.have.lengthOf(4)
+
+    const path = join(root(), 'test2', 'videos', video.uuid + '-240.mp4')
+    const probe = await audio.get(ffmpeg, path)
+      .catch(err => expect(err, 'Could not read the audio stream of the video file.').to.be.null)
```

Comment:
What's the point of this test? To be sure `audio.get` does not throw an exception?